### PR TITLE
Kb/deprecated tls

### PR DESCRIPTION
**Pull request by ker-an**
To show an error message to users on older TLS versions (v1 and v1.1) before they are deprecated.

### CODE REVIEW

File: templates/partials/top/tls-deprecation.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{{> n-messaging-client/templates/components/n-alert-banner
+	theme='error'
+	contentLong='<p><b>Warning!</b> After 18th June 2018 the web browser you are currently using won’t be able to access FT.com.</p>'
```
Comment (tavvy) [inline templates/partials/top/tls-deprecation.html]: I think you _may_ need a `contentShort` for smaller screen sizes

Comment (adgad) [reply templates/partials/top/tls-deprecation.html]: We're kind of struggling to get a meaningful message that fits in one line! Is there any particular reason the banner has a fixed height? Do you reckon we could make it wrap?

Comment (tavvy) [reply templates/partials/top/tls-deprecation.html]: Not that I'm aware of, I imagine it was a design consideration though. I believe @jonnywyatt has multiple lines in the new privacy policy message

**Comment by ker-an** [comment]
Long:

image

Short:

image

@tavvy happy last day! could you take a look at the PR and add your seal of approval?

**Pull request merged**